src/m1_facilitator/models/DepositIntent.ts

```diff
@@ -14,7 +14,6 @@
 //
 // ----------------------------------------------------------------------------
```

Comment:
these 2 lines of DepositIntent not needed

Reply:
Done